src/importexport/musicxml/internal/musicxml/export/exportmusicxml.cpp

```diff
@@ -2595,7 +2595,9 @@ void ExportMusicXml::clef(staff_idx_t staff, const ClefType ct, const String& ex
 
     int line = ClefInfo::line(ct);
     m_xml.tag("sign", info.sign);
-    m_xml.tag("line", line);
+    if (std::string(info.sign) != "TAB") {
```

Comment:
Would this also apply to the percussion clef?
(if so, you can take the opportunity to replace std::string with std::string_view.)